story_builder/location.py

```diff
@@ -32,6 +26,13 @@ def __init__(self, id, connected_areas=None):
         self.id = id
         self.connected_areas = connected_areas
 
+        # Hostiles and Friendlies may be spawned at random
+        # using the options list, or manually by pre-filling
+        # during initialization. These should be lists of
+        # instantiated NPC class objects.
+        self.hostiles = [] 
+        self.friendlies = []
```

Comment:
This took me 30 minutes to figure out.

## Problem: 

When I spawned hostiles in ANY location, they would be present in ALL locations until defeated. (as if they followed you)

## Cause: 

I don't know whether I forgot about this, or maybe I mixed in another programming language and confused myself... but when you define a variable at the highest level like I did in the link above, that is a CLASS VARIABLE, it shares a single value across all instances of that class.

Whoops.

## Solution:

Just need to move the definition within a method, so that it is unique to each object instance.

---

At least now we have an easy way to make hostiles that chase you wherever you go!